netcon_stuff/0s_defuser/0s_defuser.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+const net = require('net');
+const readline = require('readline');
+const path = require('path');
+
+const TICK_RATE = 64;
+
+let previous = {};
+let timeout10s;
+let timeout5s;
+let counter = 0;
+
+let defuseMode = 0;
+let explosionTime = 0;
+
+const sendOnConnect = `developer 1;
+con_filter_enable 2;
+con_filter_text "Time until explosion: ";
+bind 0 "echo defuse_0s_0";
+bind - "echo defuse_0s_1";
+bind = "echo defuse_0s_2";`;
+
+const onChange = data => {
+	const hasPrevious = data.Name in previous;
+	const isDifferent = !hasPrevious || previous[data.Name].In !== data.In;
+
+	if (!isDifferent) {
+		return;
+	}
+
+	if (data.Name === 'bomb_planted') {
+		previous[data.Name] = data;
+
+		if (counter === 2) {
+			explosionTime = Date.now() + 40000;
+			clearTimeout(timeout10s);
+			clearTimeout(timeout5s);
+
+			// This should be set between 29970 - 29980

```

Comment:
If you go lower than `29970` you get `0.002` defuse time, if you go higher than `29980` the bomb explodes.